wled00/data/index.js

```diff
@@ -233,7 +234,7 @@ function onLoad() {
     });
 	} else
 		loadBg(cfg.theme.bg.url);
-	loadSkinCSS('skinCss');
+	if (cfg.comp.css) loadSkinCSS('skinCss');
```

Comment:
Unfortunately using a local storage option will prevent using skinned UI on new installs.
Which is the idea behind skinning to have a distinctive look on 1st boot (prepared in advance WLED installs).

Reply:
There is no (easy) way to pre-install a `skin.css` yet.
However, I agree that a new client device should use the file by default, if available.
For now I'll enable the option by default, though I'll be looking into a way to save an unnecessary 404 request in the (probable) case a skin.css is not present.